src/en/guide/conf/config/logging.adoc

```diff
@@ -1,4 +1,4 @@
-Since Grails 3.0, logging is handled by the http://logback.qos.ch[Logback logging framework] and can be configured with the `grails-app/conf/logback.xml` file.
+Since Grails 3.0, logging is handled by the http://logback.qos.ch[Logback logging framework] and can be configured with the `grails-app/conf/logback-spring.xml` file.
 
 NOTE: Since Grails 5.1.2 support for groovy configuration (`grails-app/conf/logback.groovy`) has been removed (by logback 1.2.9). It is possible to add back groovy configuration by adding the https://github.com/virtualdogbert/logback-groovy-config[logback-groovy-config] library to your project.
```

Comment:
I think we need this library updated to Groovy 4 in order to make this claim for Grails 7.

Reply:
We may wait to see if something moves here:
https://github.com/virtualdogbert/logback-groovy-config/issues/15

Reply:
Good idea, then we can keep the Note about logback.groovy. @jamesfredley Have you tried that this actually works?

Reply:
Also, is it relevant to mention Grails 3.0 anymore? Can't we just write "Logging is handled by..."?